Makefile

```diff
@@ -42,6 +46,40 @@ GO_APIDIFF := $(TOOLS_DIR)/$(GO_APIDIFF_BIN)
 ENVSUBST_BIN := bin/envsubst
 ENVSUBST := $(TOOLS_DIR)/$(ENVSUBST_BIN)
 
+
+export GO111MODULE=on
+unexport GOPATH
+
+# Directories.
+ARTIFACTS ?= $(REPO_ROOT)/_artifacts
+TOOLS_DIR_DEPS := $(TOOLS_DIR)/go.sum $(TOOLS_DIR)/go.mod $(TOOLS_DIR)/Makefile
+
+REPO_ROOT := $(shell git rev-parse --show-toplevel)
+GH_REPO ?= kubernetes-sigs/cluster-api-provider-openstack
```

Comment:
@christopherhein thanks for the review
to do items listed https://github.com/kubernetes-sigs/cluster-api-provider-nested/issues/209#issuecomment-898166515   .. it might take some time

Reply:
Okay, great. I'll continue reviewing this.